src/core/include/openvino/op/roi_align.hpp

```diff
@@ -106,6 +106,16 @@ class OPENVINO_API ROIAlign : public Op {
     /// \param spatial_scale   Spatial scale factor used to translate ROI coordinates
     /// \param mode            Method of pooling - 'avg' or 'max'
     /// \param aligned_mode    Method of coordinates alignment - 'asymmetric', 'half_pixel_for_nn' or 'half_pixel'
+    ROIAlign(const Output<Node>& input,
```

Comment:
In general we shouldn't change operation itself especially interface without specific reason. What the reason here?

Reply:
Done